cmd/dualityd/main.go

```diff
@@ -12,7 +12,6 @@ import (
 
 func main() {
 	rootCmd, _ := cmd.NewRootCmd()
-	rootCmd.AddCommand(AddConsumerSectionCmd(app.DefaultNodeHome))
```

Comment:
won't removing this prevent the Dockerfile docker setup from working again?

Reply:
@teddyknox pointed out that the command has actually moved to a more appropriate site in `cmd/dualityd/cmd/root.go`: https://github.com/duality-labs/duality/pull/429/files#diff-71d03e33f2e287779868caecb576a089c02c6aeb350b98a97a69018bf51eb2d1R149